Packs/Campaign/ReleaseNotes/3_3_0.md

```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+
+##### Detect & Manage Phishing Campaigns
+- Fixed a critical issue where a "pepetual lock" would occur, causing all phishing incidents to eventually time-out. The perpetual lock scenario would occur when the **AutomaticallyCloseIncidents** playbook input was set to True, because incidents were closing other incidents which were obtaining the lock, thus resulting in the lock never being released.
```

Comment:
```suggestion
- Fixed a critical issue where a "perpetual lock" would occur, causing all phishing incidents to eventually time-out. The perpetual lock scenario would occur when the **AutomaticallyCloseIncidents** playbook input was set to True, because incidents were closing other incidents which were obtaining the lock, thus resulting in the lock never being released.
```